src/Appwrite/Utopia/Database/Validator/Queries/Messages.php

```diff
@@ -5,16 +5,13 @@
 class Messages extends Base
 {
     public const ALLOWED_ATTRIBUTES = [
-        'topics',
-        'users',
-        'targets',
         'providerId',
         'deliveredAt',
         'deliveredTo',
-        'deliveryErrors',
         'status',
         'description',
-        'data'
+        'providerType',
+        'message'
```

Comment:
@abnegate,

> Where does `message` come from?

Ah, right. There is no message attribute so I'll remove this.

> I think we need to keep `data` here instead

You can't filter on `data` because data is JSON/nested, though, right?

Reply:
I've updated the list of attributes to match what's available based on attribute type and what's in the collection:

https://github.com/appwrite/appwrite/blob/fb0df50bf66a3ab3d24016abc898cea2902e3fb9/app/config/collections.php#L1503-L1640